### ISSUE TITLE
How to calculate accuracy of (un)corrected reads? 

### ISSUE THREAD

**Issue by csuliweilong** [BioInf-Wuerzburg/proovread#24]
Hi @thackl,

1) Depending on your proovread paper, how to calculate the accuracy of (un)corrected reads?

2) Following your suggestion, I use mapPacBio.sh of BBMap to evaluate the correction result. When using mapPacBio.sh to map corrected reads to reference, the length of some reads exceeds the maxlength threshold(6000) of BBMap. What is your solution?

Best
Weilong

**Comment by thackl** [comment]
1) Accuracy simply is "number of matching bases" / "number of total read bases". We use read length rather than alignment length/aligned read length, as we do also want to penalize clipped ends, because clipping indicates erroneous read ends and ignoring clipped bases would overestimate accuracy. Regarding uncorrected reads, we used to placement information and the chimera status of the corrected read to align the uncorrected read with high sensitivity to the same reference location. For the paper, we ignored chimeric reads, because aligning those globally will only give proper results, if the reads is first cut at break points. This, however, is non trivial and we did not have particular interest in the specific error rate of the few chimeric reads - assuming, the accuracy won't significantly differ from other uncorrected reads

2) Use reads in FASTA format, bbmap will internally split them in 6kb chunks and will therefore be able to map reads of any length. There is a slight chance, that the split read mapping might differ from aligning the read as a whole, but I have not yet encountered such a scenario especially for corrected reads.
